call_center.py

```diff
@@ -40,49 +41,98 @@
     Price - 22$
     '''
 
+menu_ld = [{'Traditional': 19}, {'Margherita': 22}, {'Pizza Bianca': 21},
+           {'Marinara': 17}, {'Havaiian': 25}, {'Bruschetta': 22}]
+
 all_stuff = ['Hanna', 'Greg', 'Michael', 'Chloe', 'Tyler', 'Ashley', 'Jessica']
 stuff = random.choice(all_stuff)
 
 greetings = ['Hi!', 'Hello!', 'Yo!', 'Sup?', 'Whazzup?', 'Howdy!', 'Hi there!']
 hi = random.choice(greetings)
 
 misunderstanding = [
-    'What do you mean? Does it means yes or no?',
+    'What do you mean? Does it mean yes or no?',
     'Could you please repeat that?',
-    'Yes or No? You can answer in russian if you want...',
     'Sorry, I dont understand. Do you mean yes?',
-    'Not so fast cowboy! You push the wrong button ~_~',
+    'Not so fast cowboy! You push a wrong button ~_~',
     'C\'mon dude! Do you want some pizza or not?'
 ]
 
 
 # CHAT STARTS HERE>>>>
 
+print("********************************************")
 print('''
 {} My name is {}! What do I call you?
 '''.format(hi, stuff))
 usr_name = input()
 
-# print('''
-# Ok {}! Looking for some classic italian pizza?'''.format(usr_name))
-
 print('''
-Ok {}! Looking for some classic italian pizza? [Y/n (Д/н)]:
+Nice to meet you, {}! Looking for some classic italian pizza? [Y/n]:
 '''.format(usr_name))
 
 while True:
     response = input().upper()
 
-    if re.match("[YД]", response):
-        print(menu_full)
+    if re.match("[Y]", response):
+        print()
         break
-    elif re.match("[NН]", response):
+    elif re.match("[N]", response):
         print()
         print("Ok {}. See you later!".format(usr_name))
-        raise SystemExit(1)
+        exit()
     else:
         print()
         print(random.choice(misunderstanding))
         print()
 
-print("next question")
+print("Here is our short menu:")
+print()
+
+with open('pizza.json') as f:
+    short_menu = json.load(f)
+for pizzas in short_menu:
+    print(pizzas['position'], pizzas['name'])
+
+print()
+print('''
+Choose what do you like and press its number. Just one number of one pizza ^_^
+If you want more then one... Keep calm and choose your FIRST pizza '_'
+If you want to see our Full Menu with prices just type "Menu"
+When you choose enough of pizza type "Done"
+''')
+
+basket = []
+
+
+while True:
+    usrs_choice = input()
+
+    if re.match("[M,m]", usrs_choice):
+        print(menu_full)
+        continue
+
+    elif re.match("[D,d]", usrs_choice):
+        print("Ok dude! I need to write more python code to serve you")
+        break
+
+    elif re.match("[1,2,3,4,5,6]", usrs_choice):
+        for pizdict in short_menu:
+            if int(usrs_choice) == pizdict['position']:
+                flag = True
+                for u_o in basket:
+                    if u_o['name'] == pizdict['name']:
+                        u_o['quantity'] += 1
```

Comment:
This part of the code really looks like a perfect use case for [Counter ](https://docs.python.org/3/library/collections.html#collections.Counter)